src/components/calcite-notice/calcite-notice.scss

```diff
@@ -53,81 +54,76 @@
   --calcite-notice-width: 100%;
 }
 
+//TODO: more items needed here
 :host {
-  display: none;
-  text-align: left;
-  margin: 0 auto;
-  box-sizing: border-box;
+  @apply hidden
+    text-left 
+    my-0 
+    mx-auto
+    box-border
+    max-w-full
+    opacity-0
+    pointer-events-none
+    bg-foreground-1;
   width: var(--calcite-notice-width);
-  max-width: 100%;
   max-height: 0;
-  background-color: var(--calcite-ui-foreground-1);
-  opacity: 0;
-  pointer-events: none;
   transition: $transition;
   border-left: 0px solid;
   box-shadow: 0 0 0 0 transparent;
```

Comment:
Is this defined like this to accommodate a CSS transition?

Reply:
Are you talking about line $transition? That was like that. I didn't know what to do with this one. By the way, I closed this issue and opened a new one so that screener can do its thing. Please review PR here. I've made some of your suggested updates. 

https://github.com/Esri/calcite-components/pull/1633